src/binary/requests/saveSnippet.ts

```diff
@@ -0,0 +1,16 @@
+export type SaveSnippetRequest = {
+  code: string;
+  filename: string;
+  start_offset: number;
+  end_offset: number;
+};
+
+export type SuccessSaveSnippetResponse = "Success";
```

Comment:
Results should always be objects, for them to be future proof.

Reply:
how do i do that? the json response is `"Success"`